index3-1.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>課題❶-3</title>
+</head>
+<body>
+    <script>
+
+        let task = ['=====================',
```

Comment:
task の内容は変化しないので `const` でいいと思います。
```suggestion
        const task = ['=====================',
```